tests/unit_tests/test_wave_03.py

```diff
@@ -2,17 +2,17 @@
 from swap_meet.vendor import Vendor
 from swap_meet.item import Item
 
-@pytest.mark.skip
+# @pytest.mark.skip
 def test_item_overrides_to_string():
     test_id = 12345
     item = Item(id=test_id)
 
     item_as_string = str(item)
 
-    expected_result = f"An object of type Item with id {test_id}."
+    expected_result = f"An object of type Item with id {test_id}"
```

Comment:
Instead of modifying the test to match the implementation, you should modify the implementation to pass the test. So instead of removing this `.`, I'd add a `.` to `Item`.